integrations-and-supported-tools/transformers/notebooks/Neptune_Transformers.ipynb

```diff
@@ -0,0 +1,382 @@
+    "tags": [
+     "comment"
+    ]
+   },
+   "source": [
+    "---\n",
+    "\n",
+    "You now have new run in Neptune! From here on, we'll use the `run` object to log metadata.\n",
+    "\n",
+    "**To open the run in Neptune, follow the link that appeared in the cell output.**\n",
+    "\n",
+    "There's not much to display yet, but keep the tab with the run open to see what happens next."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setting up model and data for training"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from datasets import load_dataset, load_metric\n",
+    "from transformers import AutoTokenizer\n",
+    "from transformers import AutoModelForSequenceClassification, TrainingArguments, Trainer\n",
+    "from transformers.integrations import NeptuneCallback"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Loading the data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "task = \"cola\"\n",
+    "model_checkpoint = \"distilbert-base-uncased\"\n",
+    "batch_size = 16\n",
+    "dataset = load_dataset(\"glue\", task)\n",
+    "metric = load_metric('glue', task)\n",
+    "num_labels = 2"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Create Tokenizer for the model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tokenizer = AutoTokenizer.from_pretrained(model_checkpoint, use_fast=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Preprocess the dataset"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def preprocess_function(examples):\n",
+    "    return tokenizer(examples['sentence'], truncation=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "encoded_dataset = dataset.map(preprocess_function, batched=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Instantiate the model for finetuning"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model = AutoModelForSequenceClassification.from_pretrained(model_checkpoint, num_labels=num_labels)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Create the training arguments for model finetuning"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model_name = model_checkpoint.split(\"/\")[-1]\n",
+    "\n",
+    "args = TrainingArguments(\n",
```

Comment:
The tests time-out currently. Can you decrease the `num_train_epochs` to `2` and play around with other args so that the entire test takes less than 10 mins?
If that is not feasible, I'll increase the timeout duration